indico/modules/rb_new/client/js/components/TimeRangePicker.jsx

```diff
@@ -0,0 +1,198 @@
+/* This file is part of Indico.
+ * Copyright (C) 2002 - 2018 European Organization for Nuclear Research (CERN).
+ *
+ * Indico is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Indico is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Indico; if not, see <http://www.gnu.org/licenses/>.
+ */
+
+import moment from 'moment';
+import React from 'react';
+import propTypes from 'prop-types';
+import {Dropdown} from 'semantic-ui-react';
+import {Translate} from 'indico/react/i18n';
+import {toMoment} from '../util';
+
+import './TimeRangePicker.module.scss';
+
+
+const _serializeTime = time => (time ? time.format('HH:mm') : null);
+
+function _humanizeDuration(duration) {
+    const hours = duration.hours();
+    const minutes = duration.minutes();
+    if (hours === 1 && minutes === 0) {
+        return Translate.string('1 hour');
+    } else if (hours !== 0) {
+        return Translate.string('{time} hours', {time: hours + (minutes / 60)});
+    } else {
+        return Translate.string('{time} minutes', {time: minutes});
+    }
+}
+
+export default class TimeRangePicker extends React.Component {
+    static propTypes = {
+        startTime: propTypes.object,
```

Comment:
The way to go pointed out by the [`prop-types`](https://www.npmjs.com/package/prop-types) lib is importing it capitalizing the first letter i.e `PropTypes`. I know we have it imported as `propTypes` in other places of our code base but we should choose one way and be consistent all over.